pcdscalc/be_lens_calcs.py

```diff
@@ -181,37 +180,54 @@ def get_lens_set(set_number_top_to_bot, filename=None):
         before experiments, this is to specify what number set.
     filename : str, optional
         File path of the lens_set file.
-        This is a binary file generated with `numpy.save` which saves an array
-        to a binary file in NumPy .npy format.
+    get_all : bool
+        Indicates if it should return the entire stack of lens.
 
     Returns
     -------
     lens_set : list
         [numer1, lensthick1, number2, lensthick2 ...]
     """
     if filename is None and LENS_SET_FILE is None:
-        logger.error('You must provide the path to the lens_set file or you '
-                     'must configure it via :meth: `configure_lens_set_file`')
-        return
+        err_msg = ('You must provide the path to the lens_set file or you '
+                   'must configure it via configure_lens_set_file function')
+        logger.error(err_msg)
+        raise ValueError(err_msg)
     elif filename is None:
         filename = LENS_SET_FILE
     if not os.path.exists(filename):
-        logger.error('Provided invalid path for lens set file: %s', filename)
-        return
+        err_msg = ('Provided invalid path for lens set file: %s', filename)
+        logger.error(err_msg)
+        raise FileNotFoundError(err_msg)
```

Comment:
This error message is also a tuple instead of a substituted string

Reply:
should also be fixed in ef27fed